src/css/bookInput.css

```diff
@@ -0,0 +1,39 @@
+.formSection {
+  font-family: RobotoSlab, 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin: 2rem 0 0 5%;
+}
+
+.formSection > h2 {
+  font-weight: bold;
+  opacity: 0.5;
+}
+
+form {
+  display: flex;
+  flex-direction: row;
+  column-gap: 7px;
+}
+
+form > input {
+  margin: 0.5rem 0;
+  width: 30%;
+  height: 1.9rem;
+  border-radius: 2px;
+  padding: 0 1rem;
+  font-family: RobotoSlab, 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 1rem;
+  border: 1px solid  #706f6f;
+}
+
+form > button {
+  background-color: dodgerblue;
+  color: rgb(255, 255, 255);
+  padding: 4px 8px;
+  font-family: RobotoSlab, 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
Great job @danielochuba 🤝

- [ ] As stated in the project requirements, kindly ensure you do not repeat the same rule. Instead, create a single class for the `font-family` and reuse it in your JSX. Kindly take a look at this [article](https://speckyboy.com/good-bad-css-practices/) for more clarity.